src/v0/destinations/am/utils.js

```diff
@@ -55,12 +55,19 @@ function getDeviceManufacturer(payload, sourceKey) {
 }
 
 function getPlatform(payload, sourceKey) {
-  const payloadVal = get(payload, sourceKey);
-  return payload.channel
-    ? payload.channel.toLowerCase() === 'web'
-      ? 'Web'
-      : payloadVal
-    : payloadVal;
+  let payloadVal = get(payload, sourceKey);
+  if (!payloadVal) {
```

Comment:
Can this be removed now as server-side SDK's handled sending channel=server now ?

Reply:
Removed that.